app/controllers/concerns/caching_headers.rb

```diff
@@ -40,4 +50,10 @@ def build_surrogate_control(max_age, stale_while_revalidate: nil, stale_if_error
     surrogate_control += ", stale-if-error=#{stale_if_error}" if stale_if_error
     surrogate_control
   end
+
+  def fastly_enabled?
+    return if Rails.env.development?
+
+    ApplicationConfig["FASTLY_API_KEY"].presence && ApplicationConfig["FASTLY_SERVICE_ID"].presence
```

Comment:
**suggestion (non-blocking):** I think `.presence?` would be better here. Using `presence` will result in a truthy value (`nil` or the value of `ApplicationConfig["FASTLY_SERVICE_ID"]`) when we really only care about the boolean value.

Reply:
@citizen428 ah good point! i hadn't fully reviewed the code yet (hence why i left it a draft and didn't mark ready for review) since tests were failing, but this is a good call. i chatted a bit with @atsmith813 today and i think we may not even _need_ to two divergent code paths! we're going to experiment and see if this is the case (i'm updating the PR description to explain this further 😄)